site/src/pages/components/focusgroup.explainer.mdx

```diff
@@ -1223,7 +1248,7 @@ navigation modes, while a user not working with an AT (or ATs designed for visua
 would enable the full spatial navigation model. An opt-in for spatial navigation would certainly
 be a requirement and could be an extension to `focusgroup` (e.g., `focusgroup=spatial`).
 
-## 10. Open Questions
+## 11. Open Questions
```

Comment:
Can you have these link out to respective issues if they don't already exist?

Reply:
Yep!